Shared/ThreadDto.cs

```diff
@@ -0,0 +1,10 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace SharpScape.Shared.Dto;
+
+public class ThreadDto 
+{
+    [Required]
+    public Guid Id { get; set; }
+
```

Comment:
```suggestion
public class ThreadInfoDto 
{
    [Required]
    public Guid Id { get; set; }
    public string Title { get; set; }
    public int Votes { get; set; }
    public int Replies { get; set; }
    public int Views { get; set; }
```
`ThreadInfoDto` is going to need to be able to carry all relevant information about a thread to the frontend, not just the `Id`

Reply:
Right now I only use Id. I will add title and body since they are needed for create a thread. Other  information is irrelevant because once we get Id we can get other info using sql query.